### PR TITLE
Add telemetry tweet

### PR DESCRIPTION
**Pull request by chauhankaranraj**
Related to aicoe-aiops/ocp-ci-analysis#304

### CODE REVIEW

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm

File: tweets/2021-07/telemetry.tweet [hunk 1]
```diff
@@ -0,0 +1 @@
+Analyzing operational data collected from OpenShift CI clusters can help provide insights into OpenShift releases. Watch https://youtu.be/D38gR-7c2dc to learn how we use python to fetch prometheus metrics collected from CI clusters, and explore it in a jupyter notebook.
```
Comment (oindrillac) [inline tweets/2021-07/telemetry.tweet:1]: @chauhankaranraj how about tagging the accounts?

```suggestion
Analyzing operational data collected from @OpenShift CI clusters can help provide insights into @OpenShift releases. Watch https://youtu.be/D38gR-7c2dc to learn how we use python to fetch Prometheus metrics collected from CI clusters, and explore it in a Jupyter notebook.
```

Comment (chauhankaranraj) [reply tweets/2021-07/telemetry.tweet:1]: I like that idea :) Should we "@ mention" (or maybe hashtag?) prometheus and jupyter too, or just openshift?

Comment (oindrillac) [reply tweets/2021-07/telemetry.tweet:1]: sure, why not

Comment (chauhankaranraj) [reply tweets/2021-07/telemetry.tweet]: updated

**Review by oindrillac — APPROVED** [review]
/lgtm 🐦 
/hold

**Pull request merged**